.gitignore

```diff
@@ -6,6 +6,10 @@
 target/
 bin/
 
+# Ignore generated archetypes
```

Comment:
I believe convention is that this goes to `${project.build.directory}/generated-sources/`, so it should not be necessary due to `target/` exclusion.